level0/provider.tf

```diff
@@ -0,0 +1,27 @@
+provider "aws" {
+  region = "us-east-1"
+}
+
+terraform {
```

Comment:
Level0 should automate the s3 bucket creation and should have a remote state.
Delete the manually created s3 bucket from the console and recreate it here using terraform